dependency/dependency_test.go

```diff
@@ -239,6 +229,34 @@ func (c *ClientSet) createConsulTestResources() error {
 	return nil
 }
 
+func (c *ClientSet) createConsulSamenessGroups(name, partition, failoverPartition string) error {
+	members := append([]api.SamenessGroupMember{}, api.SamenessGroupMember{
+		Partition: partition,
+	}, api.SamenessGroupMember{
+		Partition: failoverPartition,
+	})
+	sg := &api.SamenessGroupConfigEntry{
+		Kind:               QuerySamenessGroup,
+		Name:               name,
+		Partition:          partition,
+		DefaultForFailover: true,
+		Members:            members,
+	}
+	log.Printf("sameness group: %+v", sg)
```

Comment:
these log lines needed?

Reply:
removed in edd0f7394427f64c6fc9de7aa74dd6bac9c32f80